index.bs

```diff
@@ -4983,6 +5138,12 @@ IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registr
     The user verification method extension returns to the [=[WRP]=] which user verification methods (factors) were
     used for the WebAuthn operation.
 - Specification Document: Section [[#sctn-uvm-extension]] of this specification
+    <br/><br/>
+- WebAuthn Extension Identifier: credProps
+- Description: This [=client extension|client=] and [=authenticator extension|authenticator=] [=registration extension=] enables reporting of a newly-created [=credential=]'s properties to the 
```

Comment:
s/ and [=authenticator extension|authenticator=] //    -- ie delete it  :)

additionally, suggest:
s/ [=credential=]'s properties / [=credential=]'s properties, as determined by the [=client=] /

Reply:
Accepted and updated.